pkg/summary/summary_test.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2019 FairwindsOps Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package summary
+
+import (
+	"testing"
+
+	"github.com/fairwindsops/goldilocks/pkg/kube"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+func TestRun(t *testing.T) {
+	fakeVpaLabels := map[string]string{
+		"kind": "Deployment",
+	}
+
+	kubeClientVPA := kube.GetMockVPAClient()
+
+	fakeVpaListOptions := metav1.ListOptions{
+		LabelSelector: labels.Set(fakeVpaLabels).String(),
+	}
+
+	_, errOk := kubeClientVPA.Client.AutoscalingV1beta2().VerticalPodAutoscalers("").List(fakeVpaListOptions)
+	assert.NoError(t, errOk)
+
+	var summary Summary
```

Comment:
Looks like we're just testing for an empty summary here?

Curious, why the call to `List`? Should we check that it returns something with `len() == 0`?

Reply:
I don't think that needs to be there. Not entirely sure. @mgoodhart5

Reply:
Great question. We fixed this! Thanks!